brainsets_pipelines/cmi_mipdb_2016/eyetracking.py

```
@@ -0,0 +1,695 @@
+    eeg_s = np.array([a[0] for a in all_anchors], dtype=np.float64)
+
+    if len(all_anchors) == 1:
+        slope = 1.0
+        intercept = float(eeg_s[0] - et_s[0])
+    else:
+        slope, intercept = np.polyfit(et_s, eeg_s, 1)
+        slope = float(slope)
+        intercept = float(intercept)
+        residuals = slope * et_s + intercept - eeg_s
+        rmse = float(np.sqrt(np.mean(residuals**2)))
+        logging.info(
```

Ruff (0.15.12)

[error] Too many arguments for `logging` format string

(PLE1205)